### ISSUE TITLE
A11y: Conferences dropdown looks like a select, doesn't quack like a select

### ISSUE THREAD

**Issue by dstorey** [smashingmagazine/redesign#115]
https://next.smashingmagazine.com/events/san-francisco-2017/

Steps:

Navigate to the conferences drop down and try to open it with keyboard

Expected:

Should open with enter or space

Actual:

Chrome/Safari/FF: can't open it at all
Edge: doesn't open with enter but does with space

Further more when you open the drop down arrows should move between items, but instead the page scrolls.

It also doesn't have select control semantics (div with no control type via role, and checkboxes inside). It would probably be better to use the aria for selects if it is not possible to use an actual select due to styling.

**Comment by vitalyfriedman** [comment]
Thanks @dstorey — we will be looking into the issue!

**Comment by vitalyfriedman** [comment]
Has been solved by Heydon Pickering. Thanks for reporting it @dstorey :-)
